Dockerfile

```diff
@@ -61,6 +61,7 @@ RUN if test "${WITH_ORACLE}" = "ON"; then \
     -DWITH_RSVG=1 \
     -DUSE_PROJ=1 \
     -DUSE_WFS_SVR=1 \
+    -DGDAL_USE_ARROW=1 \
```

Comment:
What's these build options?
I didn't find any reference in the MapServer sources...

Reply:
Arrow/Parquet support is here:
https://gdal.org/development/building_from_source.html#cmdoption-arg-GDAL_USE_ARROW